controllers/auth.controller.js

```diff
@@ -0,0 +1,53 @@
+const authService = require("../services/auth.service");
+require("dotenv").config();
+
+const registerUser = async (req, res) => {
+
+  if (JSON.stringify(req.body)==="{}") {
```

Comment:
`if (!req.body)`
if we write it like this would it create any issue?

Reply:
Yes, If I send nothing (full blank **Body** in postman), still console.log show a {} empty array not undefined or null. For that reason I used that.